config/i18n/all-langs.ts

```diff
@@ -148,28 +148,28 @@ export const i18nextCodes = {
 };
 
 // These are for the language selector dropdown menu in the footer
-export const langDisplayNames = {
-  english: 'English',
-  espanol: 'Español',
-  chinese: '中文（简体字）',
-  'chinese-traditional': '中文（繁體字）',
-  italian: 'Italiano',
-  portuguese: 'Português',
-  ukrainian: 'Українська',
-  japanese: '日本語'
-};
+export enum LangNames {
+  English = 'English',
+  Espanol = 'Español',
+  Chinese = '中文（简体字）',
+  'Chinese-traditional' = '中文（繁體字）',
+  Italian = 'Italiano',
+  Portuguese = 'Português',
+  Ukrainian = 'Українська',
+  Japanese = '日本語'
+}
```

Comment:
Capitalising these will mean the `.env` values need to be capitalised as well, I believe.

Reply:
Disregard. I'm mixing myself up oops 😓